parcel.go

```diff
@@ -15,46 +16,89 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
 
+	result, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES ($1, $2, $3, $4)", p.Client, p.Status, p.Address, p.CreatedAt)
+	if err != nil {
+		return 0, err
+	}
+	lastIndex, err := result.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	return int(lastIndex), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
 
+	row := s.db.QueryRow("SELECT number, client, status, address, created_at FROM parcel WHERE number = $1", number)
+
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
 
-	return p, nil
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+
+	return p, err
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// реализуйте чтение строк из таблицы parcel по заданному client
 	// здесь из таблицы может вернуться несколько строк
 
+	rows, err := s.db.Query("SELECT number, client, status, address, created_at FROM parcel WHERE client = $1", client)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	for rows.Next() {
+		p := Parcel{}
+		err = rows.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+		if err != nil {
+			return res, err
```

Comment:
Лучше вернуть `nil`, чтобы не было полуданных
```suggestion
			return nil, err
```